logstash-core/src/main/java/org/logstash/ackedqueue/Queue.java

```diff
@@ -139,6 +139,10 @@ public long getCurrentByteSize() {
         return this.currentByteSize;
     }
 
+    public long getCurrentPhysicallyPersistedByteSize() {
```

Comment:
@colinsurprenant frankly I'd even just go with `getPersistedSize` :
* The fact that its `bytes` can go to JavaDoc
* current is redundant, what if not the current size is this supposed to be? :)

wdyt?